app/components/courses/summary_card_component.rb

```diff
@@ -0,0 +1,226 @@
+# frozen_string_literal: true
+
+module Courses
+  class SummaryCardComponent < ViewComponent::Base
+    attr_reader :course, :search_params, :available_placements_count, :minimum_distance_to_search_location
+
+    def initialize(course:, search_params:)
+      @course = course
+      @search_params = search_params
+      @available_placements_count = course.available_placements_count
+
+      super
+    end
+
+    def title
+      govuk_link_to(find_course_path(provider_code: course.provider_code, course_code: course.course_code), class: 'govuk-link govuk-!-font-size-24') do
+        safe_join(
+          [
+            content_tag(:span, course.provider_name, class: 'app-search-result__provider-name'),
+            content_tag(:span, course.name_and_code, class: 'app-search-result__course-name')
+          ]
+        )
+      end
+    end
+
+    def location_key
+      t(".location_key.#{course.funding}", count: @available_placements_count)
+    end
+
+    def location_value
+      return t('.location_value.not_listed') if @available_placements_count.zero?
+
+      if search_by_location?
+        safe_join(
+          [
+            t(
+              '.location_value.distance',
+              school_term:,
+              distance: content_tag(:span, pluralize(course.minimum_distance_to_search_location, 'mile'), class: 'govuk-!-font-weight-bold'),
```

Comment:
the `minimum_distance_to_search_location` still doesn't exist. The idea is to return into the `SELECT` statement because we shouldn't be looping through all the placements in Ruby to calculate the minimum distance. 

This will be done when the location search is implemented.